src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -315,22 +386,25 @@ export class HaIntegrationCard extends LitElement {
                 `
               : ""}
           </div>
-          <ha-button-menu corner="BOTTOM_START">
-            <mwc-icon-button
-              .title=${this.hass.localize("ui.common.menu")}
-              .label=${this.hass.localize("ui.common.overflow_menu")}
-              slot="trigger"
-            >
-              <ha-svg-icon .path=${mdiDotsVertical}></ha-svg-icon>
-            </mwc-icon-button>
-            <mwc-list-item @request-selected="${this._handleSystemOptions}">
-              ${this.hass.localize(
-                "ui.panel.config.integrations.config_entry.system_options"
-              )}
-            </mwc-list-item>
-            ${!this.manifest
-              ? ""
-              : html`
+          ${!this.manifest
+            ? ""
+            : html`
+                <ha-button-menu corner="BOTTOM_START">
+                  <mwc-icon-button
+                    .title=${this.hass.localize("ui.common.menu")}
```

Comment:
There is no longer the `label` which I think results in no `aria-label` at runtime (did not test, just going by the docs)?

Reply:
`title` is the HTML attribute, `label` is the mwc-icon-button one. We set both.

https://github.com/material-components/material-components-web-components/tree/master/packages/icon-button#propertiesattributes

Reply:
WTH?!? How did I not see the `label` here 🤯 . Sorry for the noise...

Reply:
haha. You caught me off too, because the `.label` is not shown on the GitHub snippet here 😅